internal/protoc/protoc.go

```diff
@@ -39,7 +39,7 @@ func BinaryPath(rootDir string) (string, error) {
 		return "", err
 	}
 	version := strings.TrimSpace(string(data))
-	protocPath := filepath.Join(rootDir, fmt.Sprintf("internal/testdata/protoc/%s/bin/protoc", version))
+	protocPath := filepath.Join(rootDir, fmt.Sprintf(".tmp/cache/protoc/%s/bin/protoc", version))
```

Comment:
nit: can we use the env var here with a fallback?

Reply:
I wasn't thinking of allowing any env vars to be used. Let me tweak the `Makefile` to make that more clear -- I see it incorrectly uses `?=` in a couple of places that really should be `:=`.